src/app/about/page.tsx

```diff
@@ -0,0 +1,35 @@
+"use client";
+import { Box } from "@nypl/design-system-react-components";
+import aboutPageElements from "../data/aboutPageElements";
+import React from "react";
+
+export default function About() {
```

Comment:
This would be awesome, this is my goal for when we build our custom `Layout` component, but currently the DS provider is in the Next-given `RootLayout` (`layout.tsx`), which wraps everything, so everything has to be a client component/page. I do think it'll be easier and clearer to try to do that refactor when we get to that work rather than during this migration but I'm also happy to do it now cc @avertrees

Reply:
You can leave it for later, but glad you're already consider this.